packages/flame/lib/src/text/common/glyph_info.dart

```diff
@@ -0,0 +1,14 @@
+/// Helper class that stores dimensions of a single glyph for a spritesheet-
+/// based font.
+class GlyphInfo {
+  double srcLeft = 0;
```

Comment:
The `rst*` attributes are used in the `rstTransforms` argument for the `canvas.drawRawAtlas` function.

We do need to have proper documentation for them, but I was thinking of leaving that for later, since we still need to merge `GlyphData` and `GlyphInfo` classes, plus there would  be some reorganization of this class to support non-monospace fonts.